### ISSUE TITLE
训练Lincir的时候是否需要自己下载CIRR数据集

### ISSUE THREAD

**Issue by wqjv** [navervision/lincir#25]
提供的训练命令：
$ python -m torch.distributed.run --nproc_per_node 8 --nnodes 1 --node_rank 0 \
--master_addr localhost --master_port 5100 train_phi.py \
--batch_size 64 \
--output_dir /path/to/your_experiment \
--cirr_dataset_path /path/to/cir_datasets/CIRR \
--mixed_precision fp16 \
--clip_model_name large \
--validation_steps 1000 \
--checkpointing_steps 1000 \
--seed 12345 \
--lr_scheduler constant_with_warmup --lr_warmup_steps 0 \
--max_train_steps 20000
有--cirr_dataset_path /path/to/cir_datasets/CIRR \命令，是要下载CIRR数据集吗，如果您有该数据集，是否可以提供一个下载链接

**Comment by wqjv** [comment]
> 我们听不懂中文。请用英语询问。

Do you need to download the CIRR dataset yourself when training Lincir.
--cirr_dataset_path /path/to/cir_datasets/CIRR \ command, is it to download the CIRR dataset, and if you have that dataset, could you provide a link to download it?

**Comment by SanghyukChun** [comment]
You don't need CIRR for training. They are only for validation. You can just train your model without validation if you want.
Please check the official web site if you want to download CIRR.
https://cirr.cecs.anu.edu.au/

**Comment by SanghyukChun** [comment]
If you really want to do that, you have to fix the code.
The fastest way is to remove this part, and set the args.validation_steps to 0.
https://github.com/navervision/lincir/blob/master/train_phi.py#L166-L172
https://github.com/navervision/lincir/blob/master/train_phi.py#L275C66-L275C82

**Comment by wqjv** [comment]
> If you really want to do that, you have to fix the code. The fastest way is to remove this part, and set the args.validation_steps to 0. https://github.com/navervision/lincir/blob/master/train_phi.py#L166-L172 https://github.com/navervision/lincir/blob/master/train_phi.py#L275C66-L275C82

Thank you very much. I'll try it.
